docker/pg-entrypoint/00-create-pg-db.sh

```diff
@@ -7,6 +7,9 @@ log_statement = 'all'
 log_disconnections = off
 log_duration = on
 log_min_duration_statement = -1
+shared_preload_libraries = 'pg_stat_statements'
+track_activity_query_size = 2048
+pg_stat_statements.track = all
```

Comment:
It might be a good idea to add `pg_stat_statements.max = 10000` (default: 5000) to ensure some statements are not discarded (e.g. least executed statements that may hold valuable information for the analytics during long scenario execution).

Reply:
updated